code/modules/mob/living/bloodcrawl.dm

```diff
@@ -175,5 +180,5 @@
 		for(var/obj/item/bloodcrawl/BC in C)
 			BC.flags_1 = null
 			qdel(BC)
-	QDEL_NULL(holder)
+	SEND_SIGNAL(src, COMSIG_BLOODCRAWL_DELETE)
```

Comment:
While I see where you were going here, signals are supposed to be generic event reports, not specific ones. They shouldn't substitute proc calls directly.

A simple fix to this would be to convert it as a `COMSIG_LIVING_PHASE_IN` signal, right before `forceMove(B.loc)`